### DISCUSSION TITLE
Spock 2.0 usage in 1.16.0

### DISCUSSION THREAD

**Discussion by musketyr** [testcontainers/testcontainers-java#4554] [General]
If you are Spock user and you are upgrading to `1.16.0` then be sure that you have upgraded to Spock 2.0 before otherwise the Spock tests will no longer be discovered. `org.testcontainers:spock:1.16.0` depends on Spock 2.0 which means that it indirectly introduces JUnit 5 so you need to add 

```
test {
    useJUnitPlatform()
}
```

To re-enable the tests again.

**Comment by kiview** [comment]
This is the corresponding PR for activating Spock in our own test-suite:
https://github.com/testcontainers/testcontainers-java/pull/4568
